GiniSwitchSDK/GiniSwitchSDK/Classes/ExtractionCollection.swift

```diff
@@ -8,34 +8,92 @@
 
 import UIKit
 
-public class ExtractionCollection:BaseApiResponse {
-
-    public var extractions:[Extraction]
+public class ExtractionCollection: Codable {
```

Comment:
In this case I'm not sure if you want to allow them to extend this class. In case that you don't want, you know what is missing.